src/remember.ts

```diff
@@ -0,0 +1,71 @@
+import { Operator, StrictSink } from './types';
+
+export function remember<T>(
+  numRemembered: number = 1,
+  shouldUnsubscribe: boolean = false
+): Operator<T, T> {
+  return source => {
+    const marker = {};
+    let sinks: Array<StrictSink<T> | undefined> = [];
+    let talkback: any;
+    let lasts: T[] = Array(numRemembered).fill(marker);
+    let curr = -1;
+    let started = false;
+
+    const mkTalkback = (sink: StrictSink<T>) => () => {
+      sinks[sinks.indexOf(sink)] = void 0;
+
+      if (shouldUnsubscribe) {
+        // Allow others to subscribe in the same iteration of the JS event loop
+        queueMicrotask(() => {
+          if (sinks.every(x => x === undefined)) {
+            sinks = [];
+            console.log('end');
```

Comment:
Oh, here's a console.log, probably not intended?

Reply:
oops